src/components/button/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
-  selector: 'rio-button',
+  selector: 'sg-rio-button',
```

Comment:
It has a rule which says each component should be prefixed by something. https://angular.io/docs/ts/latest/guide/style-guide.html#!#style-02-07